src/Gui/Dialogs/DlgAddPropertyVarSet.cpp

```diff
@@ -533,34 +739,61 @@ void DlgAddPropertyVarSet::clearFields()
     setOkEnabled(false);
 }
 
+void DlgAddPropertyVarSet::addDocumentation() {
+    /* Since there is no check on documentation (we accept any string), there
+     * is no signal handler for the documentation field.  This method updates
+     * the property that is being added with the text inserted as
+     * documentation/tooltip.
+     */
+
+    std::string group = comboBoxGroup.currentText().toStdString();
+    std::string doc = ui->lineEditToolTip->text().toStdString();
+
+    if (propertyItem == nullptr) {
+        // If there is no property item, we cannot add documentation.
+        return;
+    }
+
+    App::Property* prop = propertyItem->getFirstProperty();
+    if (prop == nullptr) {
+        return;
+    }
+
+    varSet->changeDynamicProperty(prop, group.c_str(), doc.c_str());
```

Comment:
```suggestion
    if (App::Property* prop = propertyItem->getFirstProperty()) {
        varSet->changeDynamicProperty(prop, group.c_str(), doc.c_str());
    }
```

But I'm not so sure about that one too - it is shorter, but it seems for me that main execution path is when `prop != null` and so it should be the least indented one.

Reply:
Indeed, I left it as is.